AggregateMetrics/AggregateMetrics.Tests/AzureWebApp/AzureWebAppTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace webAppTest
+{
+    using System;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using System.Diagnostics;
+    [TestClass]
+    public class UnitTestAzureWeb
+    {
+        [TestMethod]
+        public void TestGetValueAndReset()
+        {
+            PerformanceCounterGauge testingGage = new PerformanceCounterGauge();
+
+            var mt= testingGage.GetValueAndReset();
```

Comment:
nitpick: This variable name and lack of type information makes this unreadable without context.